beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -197,7 +191,10 @@ impl<E: EthSpec, S: Store<E>> Migrate<S, E> for BlockingMigrator<S> {
         old_finalized_block_hash: SignedBeaconBlockHash,
         new_finalized_block_hash: SignedBeaconBlockHash,
     ) {
-        if let Err(e) = S::process_finalization(self.db.clone(), state_root, &new_finalized_state) {
+        if let Err(e) =
+            self.db
```

Comment:
`self.db.process_finalization(self.db.clone(), ...)` seems like an odd pattern.

Reply:
Yeah, I cut a corner on this one.  I should be able to correct it if the rest of the PR is fine.